ddtrace/contrib/azure.functions/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""
+The foo integration instruments the bar and baz features of the
```

Comment:
We need to update this docs page.

```suggestion
The azure function integration instruments the ..... and .... features of the
```